src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -141,7 +141,12 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                                     return <div className="has-background-dark has-text-white">{row.value}</div>;
                                 }
                                 if (vulnerableVotes && row.original.partyCode === vulnerableVotes.winner.partyCode) {
-                                    return null;
+                                    return (
+                                        <span className="icon">
+                                            {" "}
```

Comment:
Why are you printing a space in addition to the symbol?

Reply:
I think TSLint did it

Reply:
It is not added when I save the file or anything. Is TSLint giving you an error that it cannot find the config file ./vscode/tslint.json?

Reply:
`git commit -m "Indicate winner in SingleDistritct by trophy"`
husky > npm run -s precommit (node v13.9.0)

🔍  Finding changed files since git revision c9c6386.
🎯  Found 2 changed files.
✍️  Fixing up src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx.
✅  Everything is awesome!

[indicate-winner-in-SingleDistrict-view 10df1cf] Indicate winner in SingleDistritct by trophy
 2 files changed, 8 insertions(+), 12 deletions(-)

Reply:
How strange. @simendag Do you know why husky would add in that space?

Reply:
{
    "extends": ["tslint:latest", "tslint-react", "tslint-config-prettier"],
    "rules": {
        "ordered-imports": false,
        "no-submodule-imports": true,
        "object-literal-sort-keys": false,
        "member-access": false,
        "interface-name": false,
        "no-object-literal-type-assertion": false,
        "array-type": false,
        "curly": true,
        "no-console": false,
        "no-shadowed-variable": false,
        "unified-signatures": true,
        "radix": false,
        "cyclomatic-complexity": true,
        "forin": false
    }
}

Reply:
> How strange. @simendag Do you know why husky would add in that space?

No idea, but might be related to the <span> being used. I would have just printed the text, a fragment, or a div (and testing them in that order).

Reply:
I still feel like this is a complete non-issue and we should just merge it. Add it as an issue for fixing later if it's very annoying.